static/app/debug/notifs/data.ts

```diff
@@ -0,0 +1,147 @@
+import type {NotificationCategory} from 'sentry/debug/notifs/types';
+
+/** TODO(ecosystem): Use actual notification platform items, maybe via API? */
+export const notificationCategories: NotificationCategory[] = [
```

Comment:
Copied this from our notification-settings page, just to layout the appearance with a whole bunch of data, this file will likely be deleted and replaced with an API call to an endpoint we have to build.